### PR TITLE
Prepare keycloak migration and some minor improvements

### PR DESCRIPTION
**Pull request by carlesarnal**

### CODE REVIEW

File: distro/openshift/auth/Dockerfile [hunk 1]
```diff
@@ -14,14 +14,10 @@ ENV PROXY_ADDRESS_FORWARDING=true
 
 USER jboss
 
-ADD realm.json /tmp/apicurio-realm.json
-ADD auth-entrypoint.sh /opt/jboss/auth-entrypoint.sh
```
Comment (EricWittmann) [inline distro/openshift/auth/Dockerfile:18]: How is the realm added on startup for a new install with these removed?

**Review by EricWittmann — CHANGES_REQUESTED** [review]

File: distro/openshift/auth/Dockerfile (hunk 1 above)
Comment (carlesarnal) [reply distro/openshift/auth/Dockerfile]: Realm import added

File: distro/openshift/auth/Dockerfile [hunk 2]
```diff
@@ -14,14 +14,10 @@ ENV PROXY_ADDRESS_FORWARDING=true
...
 COPY themes/apicurio /opt/jboss/keycloak/themes/apicurio
 
 ENV APICURIO_KEYCLOAK_USER=admin
 ENV APICURIO_KEYCLOAK_PASSWORD=password123!
 ENV APICURIO_UI_URL=http://localhost:8080/
```
[the remark this answers is not in the document]
Comment (carlesarnal) [reply distro/openshift/auth/Dockerfile]: Added a modified version of the entrypoint.

**Comment by carlesarnal** [comment]
@EricWittmann this is now ready for final review. The problem was that the change_database script was moved to a different location in Keycloak 14. The database will be automatically updated by Liquibase once we start a Keycloak 14 server against our database. I'm going to finish testing our Keycloak extensions and then we will be ready for the migration to Keycloak 14.

**Review by EricWittmann — APPROVED** [review]

**Pull request merged**